pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -591,6 +594,18 @@ func (br *OVSBridge) GetOFPort(ifName string) (int32, Error) {
 		}},
 		Where: [][]interface{}{{"name", "==", ifName}},
 	})
+	if waitUntilValid {
+		tx.Wait(dbtransaction.Wait{
+			Table:   "Interface",
+			Timeout: uint64(defaultGetPortTimeout / time.Millisecond), // The unit of timeout is millisecond
+			Columns: []string{"ofport"},
+			Until:   "!=",
+			Rows: []interface{}{map[string]interface{}{
+				"ofport": []interface{}{"set", []int32{-1}},
+			}},
+			Where: [][]interface{}{{"name", "==", ifName}},
+		})
```

Comment:
how about it just replaces the value in `Rows` when `waitUntilValid` is true, instead of making two calls?

Reply:
Updated.